lambda/test/metric_ingester.spec.ts

```diff
@@ -2,6 +2,7 @@ import { CloudWatchClient, GetMetricDataCommand, StatusCode } from '@aws-sdk/cli
 import { InvokeCommand, LambdaClient } from '@aws-sdk/client-lambda'
 import { type Context } from 'aws-lambda'
 import { mockClient } from 'aws-sdk-client-mock'
+import 'aws-sdk-client-mock-jest'
```

Comment:
I added this per [the docs](https://github.com/m-radzikowski/aws-sdk-client-mock#jest-matchers) to fix [this error](https://github.com/byu-oit/terraform-aws-humio-logger/actions/runs/4302795873/jobs/7501740593#step:7:23). If there's a better way, feel free to fix it.